pkgs/data/icons/catppuccin-papirus-folders/default.nix

```diff
@@ -0,0 +1,56 @@
+{
+  stdenvNoCC,
+  lib,
+  fetchFromGitHub,
+  gtk3,
+  papirus-icon-theme,
+  flavor ? "mocha",
+  accent ? "blue",
+  ...
+}: let
+  validAccents = ["blue" "flamingo" "green" "lavender" "maroon" "mauve" "peach" "pink" "red" "rosewater" "sapphire" "sky" "teal" "yellow"];
+  validFlavors = ["latte" "frappe" "macchiato" "mocha"];
+  pname = "catppuccin-papirus-folders";
+in
+  lib.checkListOfEnum "${pname}: accent colors" validAccents [ accent ]
+  lib.checkListOfEnum "${pname}: flavors" validFlavors [ flavor ]
+
+  stdenvNoCC.mkDerivation {
+    inherit pname;
+    version = "unstable-2022-12-04";
+
+    src = fetchFromGitHub {
+      owner = "catppuccin";
+      repo = "papirus-folders";
+      rev = "1a367642df9cf340770bd7097fbe85b9cea65bcb";
+      sha256 = "sha256-mFDfRVDA9WyriyFVzsI7iqmPopN56z54FvLkZDS2Dv8=";
+    };
+
+    nativeBuildInputs = [ gtk3 ];
+
+    postPatch = ''
+      patchShebangs ./papirus-folders
+    '';
+
+    installPhase = ''
+      runHook preInstall
+      mkdir -p $out/share/icons
+      cp -r --no-preserve=mode ${papirus-icon-theme}/share/icons/Papirus* $out/share/icons
+      cp -r src/* $out/share/icons/Papirus
+      cp -r src/* $out/share/icons/Papirus-Dark
```

Comment:
It looks like `Papirus-Dark` uses symbolic links in all of the places this would copy stuff to. So just copying this theme at all should be sufficient and you shouldn't need to duplicate it again (since it links to `../Papirus/[size]/places` anyway)

```suggestion
```

Reply:
ah okay

Reply:
Good catch that it was missing Papirus-Dark though! That last push was just failing to build for me without removing this line, and it was printing errors like this:

```
catppuccin-papirus-folders-unstable> installing
catppuccin-papirus-folders-unstable> cp: cannot overwrite non-directory '/nix/store/crq0b2nv7skpi99lq33bjwc0n2y8fyzk-catppuccin-papirus-folders-unstable-2022-12-04/share/icons/Papirus-Dark/22x22/places' with directory 'src/22x22/places'
catppuccin-papirus-folders-unstable> cp: cannot overwrite non-directory '/nix/store/crq0b2nv7skpi99lq33bjwc0n2y8fyzk-catppuccin-papirus-folders-unstable-2022-12-04/share/icons/Papirus-Dark/24x24/places' with directory 'src/24x24/places'
catppuccin-papirus-folders-unstable> cp: cannot overwrite non-directory '/nix/store/crq0b2nv7skpi99lq33bjwc0n2y8fyzk-catppuccin-papirus-folders-unstable-2022-12-04/share/icons/Papirus-Dark/32x32' with directory 'src/32x32'
catppuccin-papirus-folders-unstable> cp: cannot overwrite non-directory '/nix/store/crq0b2nv7skpi99lq33bjwc0n2y8fyzk-catppuccin-papirus-folders-unstable-2022-12-04/share/icons/Papirus-Dark/48x48' with directory 'src/48x48'
catppuccin-papirus-folders-unstable> cp: cannot overwrite non-directory '/nix/store/crq0b2nv7skpi99lq33bjwc0n2y8fyzk-catppuccin-papirus-folders-unstable-2022-12-04/share/icons/Papirus-Dark/64x64' with directory 'src/64x64'
```